src/Spago/CLI.hs

```diff
@@ -148,6 +148,7 @@ parser = do
     usePsa       = bool UsePsa NoPsa <$> CLI.switch "no-psa" 'P' "Don't build with `psa`, but use `purs`"
     openDocs     = bool NoOpenDocs DoOpenDocs <$> CLI.switch "open" 'o' "Open generated documentation in browser (for HTML format only)"
     noComments   = bool WithComments NoComments <$> CLI.switch "no-comments" 'C' "Generate package.dhall and spago.dhall files without tutorial comments"
+    tag          = CLI.optional $ CLI.optText "tag" 't' "Optional package set tag to be used instead of the latest one."
```

Comment:
I wouldn't like to have a shorthand for this flag (as we found `t` to be clashing with other flags several times already), so let's use optparse-applicative directly here:

```suggestion
    tag          = CLI.optional $ Opts.strOption (Opts.long "tag" <> Opts.help "Optional package set tag to be used instead of the latest one.")
```